packages/cli/bin/backstage-cli

```diff
@@ -19,7 +19,11 @@ const path = require('path');
 
 // Figure out whether we're running inside the backstage repo or as an installed dependency
 const isLocal = require('fs').existsSync(path.resolve(__dirname, '../src'));
-if (!isLocal) {
+
+// This is used for e2e-tests where we create a new app in a tmp folder
+const isTemp = path.resolve(__dirname).includes(require('os').tmpdir());
```

Comment:
Do we really need this if we already switch based on the env var?

Reply:
Ah, no probably not. Good catch.